data-common.json

```diff
@@ -25,11 +25,13 @@
   },
   "firefox": {
     "developer": {
+      "versions": ["firefox52", "firefox53"],
```

Comment:
So shouldn't version numbers only go in https://github.com/kangax/compat-table/blob/gh-pages/environments.json ?